README.md

```diff
@@ -5,11 +5,12 @@
 Okta ASP.NET middleware
 ========================
 
-* [Release status](#release-status)
-* [Need help?](#need-help)
-* [What you need](#what-you-need)
-* [Getting started](#getting-started)
-* [Contributing](#contributing)
+- [Okta ASP.NET middleware](#okta-aspnet-middleware)
+  - [Release status](#release-status)
+  - [Need help?](#need-help)
+  - [What you need](#what-you-need)
+  - [Getting Started](#getting-started)
+  - [Contributing](#contributing)
```

Comment:
Are these lines supposed to be indented?

Reply:
I'll double checked.